### ISSUE TITLE
Question about KeyError when visualize voxel with predictions

### ISSUE THREAD

**Issue by Ha-coding-user** [PRBonn/semantic-kitti-api#148]
Hello, I meet one Error... when use visualize_voxel.py

First, I tell you my situation
Now, I reproducing one model and extract test prediction and check in server but performance is too low...
So, I visualize these predictions and now i meet error like this

![image](https://github.com/PRBonn/semantic-kitti-api/assets/110831304/267cf318-fbb5-4136-98a2-4387edd42dc1)
when i select predictions visualizer is turn off...
and show KeyError: 'input'

> /kitti/dataset/
>           └── sequences/
>                   ├── 11/
>                   │   ├── poses.txt
>                   │   ├── image_2/
>                   │   ├── image_3/
>                   |   ├── voxels/
>                   |   |     ├ 000000.bin
>                   |   |     ├ 000005.bin
>                   │   └── predictions/
>                   │         ├ 000000.label
>                   │         └ 000005.label
>                   .
>                   .
>                   .
>                   └── 21/

This is my File Tree structure

Please reply me... thank you

**Comment by jbehley** [comment]
sorry for the late answer; I looked into the code and I only checked the visualization option if the "labels" are selected. Now, I have updated the checkboxes and the current state of change of the subdirectories.

So it should now work as intended; if you change to predictions it will automatically change to "labels" instead of trying to visualize the "input". Hope that it helps with debugging.


Let me know if it works now for you; I also will try to answer more quickly. :/
